play/apps/game/urls.py

```diff
@@ -5,10 +5,10 @@
 urlpatterns = [
     url(r'^games/$', method_dispatch(
         GET=views.games.index,
-        POST=views.games.create,
```

Comment:
Removing this removes a weird URL redirection pattern I was seeing when encountering an invalid form.

Reply:
`/games/new` => invalid url => `/games/` (same form page, but showing errors)